client/src/components/ProfilePasswordChange/index.tsx

```diff
@@ -4,14 +4,10 @@ import { renderField } from '../CommonComponents/ReduxFormFields';
 
 import React from 'react';
 import { connect } from 'react-redux';
-import { Form, Row, Label, Button, Spinner, FormProps } from 'reactstrap';
+import { Form, Row, Label, Button, FormProps } from 'reactstrap';
 import { Field, reduxForm } from 'redux-form';
 import { Link } from 'react-router-dom';
 
-interface IStateToProps {
-    changeProfilePassword: any; // FIXME
```

Comment:
lol
I believe we requested  to change it to TODO

Reply:
That's the consequence of recently set `no-unused-variables` tslint rule. This particular interface is not used yet.